src/main/java/uk/co/orchardsystems/cinema_ticketing/controller/TicketServicesController.java

```diff
@@ -0,0 +1,76 @@
+package uk.co.orchardsystems.cinema_ticketing.controller;
+
+import org.json.JSONException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.co.orchardsystems.cinema_ticketing.CinemaService;
+import uk.co.orchardsystems.cinema_ticketing.Exceptions.NoScreeningsFoundException;
+
+@RestController
+@RequestMapping("/ticketing")
+@CrossOrigin(origins = {"${frontend.url}"})
+public class TicketServicesController {
+
+    @Autowired
+    private CinemaService cinemaService;
+
+    @GetMapping("/cinemas")
+    public ResponseEntity<String> getCinemas(){
+        String cinemas = "";
+        HttpStatus response = HttpStatus.OK;
```

Comment:
The internals of these 4 methods are virtually identical. Would be nice as an exercise to extract the core logic as a lambda that accepts the specific service get method as a param.

e.g. An example, I'll leave the hard work to you :)
```
public ResponseEntity<String> getMoviesAtCinema(@RequestParam("cinemaId") int cinemaId){
    return get(CinemaService::getMoviesList, cinemaId);
}

// reuseable get method
public static ResponseEntity<String> get(serviceRequest, int... args) {
    String entities = "";
    HttpStatus response = HttpStatus.OK;
    try {
        entities = serviceRequest(int[0]);
    } catch (JSONException e) {
        e.printStackTrace();
        response = HttpStatus.NOT_FOUND;
    }
    return new ResponseEntity<>(entities, response);
}  
```

Reply:
Yeah I noticed it too, ill give this a read